web-common/src/features/templates/markdown/Markdown.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+  import DOMPurify from "dompurify";
+  import { marked } from "marked";
+  import { MarkdownProperties } from "@rilldata/web-common/features/templates/types";
+  import { V1ComponentSpecRendererProperties } from "@rilldata/web-common/runtime-client";
+
+  export let rendererProperties: V1ComponentSpecRendererProperties;
+
+  $: kpiProperties = rendererProperties as MarkdownProperties;
+  $: css = kpiProperties.css || {};
```

Comment:
`kpiProperties` should be renamed to `markdownProperties`

Reply:
Opps! Fixed!